fdbserver/fdbserver.actor.cpp

```diff
@@ -1107,6 +1108,23 @@ struct CLIOptions {
 private:
 	CLIOptions() = default;
 
+	void parseEnvInternal() {
+		for (std::string knob : getEnvironmentKnobOptions()) {
+			std::vector<std::string> kv;
+			boost::split(kv, knob, [](char c) { return c == '='; });
```

Comment:
I believe we support string knobs, in which case the `=` may be a legal character in the value. Maybe we could split only on the first `=` character and then assert that we found one.

Reply:
It's also possible (at least for strings) that the knob value is the empty string. We may not be able to require that the length of the value is non-zero then.

Reply:
Ok